data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/SimpleCrac.java

```diff
@@ -362,6 +363,7 @@ public void addCnec(Cnec cnec) {
         }
     }
 
+    @JsonSerialize(contentUsing = RangeActionSerializer.class)
```

Comment:
shouldn't the @JsonSerialie be removed for both RA and NA ?

Reply:
ok